README.md

```diff
@@ -450,6 +450,19 @@ In alphabetic order.
     </td>
   </tr>
 
+  <tr>
+    <td>
+      <img width="800px" src="https://tnl-project.org/wp-content/uploads/2018/10/logo-sketches-05.png">
+    </td>
+    <td>
+       <h1>TNL</h1>
+       <p> TNL is a collection of building blocks that facilitate the development of efficient numerical solvers and HPC algorithms. It is implemented in C++ using modern programming paradigms in order to provide a flexible and user-friendly interface such as the STL library, for example. TNL provides native support for modern hardware architectures such as multicore CPUs, GPUs, and distributed systems, which can be managed via a unified interface. </p>
```

Comment:
@TomasOberhuber please edit this info as you wish. 
The url to the logo https://tnl-project.org/wp-content/uploads/2018/10/logo-sketches-05.png has been adopted from your website but there is something wrong since it does not even appear on your own repo. Look into that.

Reply:
update: Logo renders well. 
<img width="1005" alt="Screenshot 2023-02-03 at 12 48 25 PM" src="https://user-images.githubusercontent.com/5192965/216672204-1797c23d-fd48-47d7-a28b-0edf74f17123.png">

Reply:
@henrykironde I have changed the hours to 350 on the wiki page - https://gitlab.com/tnl-project/tnl/-/wikis/GSoC-2023 . Could you, please, change the link. On the main page ( https://github.com/numfocus/gsoc/blob/master/2023/ideas-list.md ), there still the original link.

Reply:
Am not sure what you mean. The links in the PR seem fine to me.

Reply:
On the main page ( https://github.com/numfocus/gsoc/blob/master/2023/ideas-list.md ) we have a link on a file in our repo ( https://gitlab.com/tnl-project/tnl/-/blob/main/GSoC_2023.md ) but we would like to change to our wiki, i.e. https://gitlab.com/tnl-project/tnl/-/wikis/GSoC-2023. I am sorry for that. Thank you, Tomas.

Reply:
@TomasOberhuber make all the changes to this branch please

Reply:
@henrykironde Sorry, I checked the links in this branch and they are ok.